python/test/unit/function/test_interpolation.py

```diff
@@ -21,10 +21,10 @@
                   CellType.quadrilateral, CellType.hexahedron])
 
 
-def random_point_in_cell(cell_type):
+def random_point_in_reference(cell_type):
```

Comment:
I think tests should be deterministic.

Reply:
I've added `random.seed` to the tests, which I think does this